config/credentials/staging.yml.enc

```diff
@@ -1 +1 @@
-GxV4+cCT2EhX+DeLCpCIbuYhJbc5a+eQV0GAmiW2DOPGrVGdYlIPZNjys3c8B9WdijJ8iJ2znb6LXeHZo0E6yCX7Lql4T/F5fphtFVENdm3YyYNqsokDSsfMqWjatvjLVVFv52foMCZeg1P4wqwX5DxUPHyNt5dddMNxzO6IXzIQcrdUQsrfHsFGPGwDBD8hZzEA9EMLqNXqcdDk5I1gz2qbgWSYwl6CCcuSsPdYXOPvxGuHXEJt/ZMf3LHa8i3PjkUfwCXZhWhkHEJIgd0TVhx6wXTC/4hKJ+TSih1NY7iRTcm/irdzjnwy4WM4Li37rh9Aomc2+1CqIrSd8K2B88R2b65ANQtc1OY7Lot7LYcj7xxQ2j23gfHCkILHsHoJYQX55rUQ5EvvrOMNqY0TFLcm5lUahd4ddmqjysG/72b/byNgTnDG7bV10JHJVMnQbnf/0HfueBNK5541Tysnir6shg==--8Pa3LRE+KTP58DN+--ByImwOBFUzTzU7hScwTwZg==
\ No newline at end of file
+VeadEsgtu8ntoMlXqmx00azIv//SzjkvLIgqfpxc2H2Wqy3Q5CE3nONXqKkMvJoUOzRUsAW/Sax8hVu7JlFGbncvptwqyeopDNOGECxWa30C49JIZzSr43H7lelfLJEyiHkTQfgNPwbNfMSOAN61+qTQyq95vSN824Ge1a01wLFgn2Q5tXk4uY7TuCNlZ+yuOo4taVVgs1ZoM9AMwVnMgcRBDZBRhRx4g1i0E/U9Sbn5lLO5oc/dfJDlUKEHP2GndgsAvcEDCTP0PtJBslCGmctDrD8i3IGhWdHmoEKS0uRUu+dG9B86XIHsTLLpP3FQ3psvIvpYybA2c0duwteYxvmrf45PqX2Hh0DbsYe/PUdbVEj2X06SoVl9CEwtAmE+R6Wneuy61pgludgyYbZI7N/lIykBXnb1NDu/sA1C/OE7FPhCkTYF3vjWTbvbMAw6h7q3DnwzvkIKEQ7YMZvZZNpRNqwd+G8f63g1EbeHaQlrzgMkDTZ1PNr+7G6hDVo+T4JWne/AdAtOnjJJhHQZ2cxgnz9Dre4hW/RuB+f+xFUwSDs8+LTN0EV1Cw==--TlVBK2ISug9NTtUs--xfWKcOPuHGXdZa4nAOFfzQ==
```

Comment:
Why is this different from the original?

Reply:
I was working with @miao-bpl to add a `deploy` section to the credentials that could be used to configure things like username, server address, deploy path, etc. It's still a work in progress.